FoodieFun/src/App.js

```diff
@@ -1,23 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Form1 from './Components/Login';
```

Comment:
Not essential but its good practice to have more descriptive names for your components, register or registerForm would be good name examples.